src/yachtzee.ts

```diff
@@ -1042,28 +1021,46 @@ export function bestYachtzeeFamiliar(): Familiar {
     )
     .sort(
```

Comment:
is there a reason we aren't sorting by leprechaunMultiplier? it seems easier. I know that the cactus bud slightly messes with it, but frankly its effect on the sqrt term is small enough at the weights we're dealing with.
It's alright to use numericModifier, but you for sure want to slice this out into a function because you are repeating yourself a lot

Reply:
As asmo has pointed out, this could go up to a 50%meat difference. That's 1k/spleen, which could potentially mean the difference between gregging and chaining, so the term does matter to some degree\*.

\*The estimates aren't *that* accurate yet - the current estimate slightly underestimates our yachtzee profitability, but it could potentially matter when we refine it in the future.